README.rst

```diff
@@ -78,7 +94,7 @@ To run a celery worker:
 
 .. code-block:: bash
 
-    cd sprint_management_application
+    cd sprints
     celery -A config.celery_app worker -l info
 
 Please note: For Celery's import magic to work, it is important *where* the celery commands are run. If you are in the same folder with *manage.py*, you should be right.
```

Comment:
@Agrendalath, in the following section `Sentry` is mentioned. Are we planning to use it instead of our hosted infrastructure that we use to monitor logs? If not, can we remove it from the documentation and also from the places it is used?

Reply:
I find Sentry really handy for catching the full context of an error. It's currently set up for the server (I'll put credentials in the Vault), but I can remove it if we won't want to use it.